src/20231212_AaveV3Ethereum_ContinuousSecurityProposalAaveCertora/ContinuousSecurityProposalAaveCertora.md

```diff
@@ -8,8 +8,7 @@ discussions: "https://governance.aave.com/t/arfc-continuous-security-proposal-aa
 
 We offer to extend the existing [formal verification and manual review services](https://governance.aave.com/t/security-and-agility-of-aave-smart-contracts-via-continuous-formal-verification/10181) and provide additional security services:
 
-1. Governance proposal reviews - Review every governance proposal in the on-chain stage as suggested by BGDLabs.
-2. We will assist Aave and BGD in reviewing Immunify bug bounties per request.
+1. We will assist Aave and BGD in reviewing Immunify bug bounties per request.
```

Comment:
1. Governance proposal reviews - Review every governance proposal in the on-chain stage as suggested by BGDLabs.
2. We will assist Aave and BGD in reviewing Immunefi bug bounties per request.